lib/reactions/fiat_rate_update.dart

```diff
@@ -27,7 +27,9 @@ Future<void> startFiatRateUpdate(AppStore appStore, SettingsStore settingsStore,
           if (appStore.wallet!.type == WalletType.haven) {
             await updateHavenRate(fiatConversionStore);
           } else {
-            fiatConversionStore.prices[appStore.wallet!.currency] = await FiatConversionService.fetchPrice(
+            fiatConversionStore.prices[appStore.wallet!.currency] =
+            settingsStore.shouldDisableFiat ? 0.0
+                : await FiatConversionService.fetchPrice(
```

Comment:
Here no case for disable for fetch rate update for haven. I would to force stop any calls for updating the rate for haven or another wallet types above on call stack and reset price for all currencies to 0.

Reply:
So better check is fiat disabled above in timer callback function and if it's disabled then reset prices and return from the function and below keep fetching as it was before.

Reply:
@mkyq @OmarHatem28 Didn't find any use for `fiat rate_update.dart`, fiat rate is no longer updated by timer. Do we need to restore this feature?
Currently `FiatConversionService` is used in `startCurrentFiatChangeReaction` and `startCurrentWalletChangeReaction`.